tests/panic_safely.rs

```diff
@@ -44,7 +52,10 @@ fn should_panic_safely() {
 
     // Invoke `db.panic_safely() without having set `db.one`. `db.one` will
     // default to 0 and we should catch the panic.
-    let result = panic::catch_unwind(AssertUnwindSafe(|| db.panic_safely()));
+    let result = panic::catch_unwind(AssertUnwindSafe({
```

Comment:
I mentioned this in Zulip, but I did not originally intend for this test case to test parallel behavior. I'm okay with the change as is, but this will be covered by the new test in _tests/parallel/true_parallel.rs_ after these branches merge anyways.

This was meant to be the minimal working example of a safe panic.

Reply:
@kleimkuhler yeah — makes sense. The question is whether we intend to the database to be "unwind safe" (that is, recoverable within a single thread). I don't normally like to promise that, so in a way the test is testing something we don't necessarily expect to work. *On the other hand,* I can't see any reason *not* to support this scenario (but we don't presently).

Reply:
I'll merge with this test but let's update it =)